app/design/frontend/base/default/template/boltpay/connectjs.phtml

```diff
@@ -22,6 +22,35 @@ $isAllowedConnectJs = $this->isAllowedConnectJsOnCurrentPage();
 <?php if ($isAllowedConnectJs): ?>
     <?php $publishableKey = $this->getPublishableKeyForRoute(); ?>
     <?php if ($publishableKey): ?>
+        <script>
```

Comment:
This seems kinda unrelated to PPC? What does this do?

Reply:
@adentong This is related and required for PPC.  I've updated the comment below to be more clear.